scripts/prep-key.sh

```diff
@@ -1,12 +1,5 @@
 #!/bin/sh
 set -e
 
-export DEPLOY_BRANCH=${DEPLOY_BRANCH:-master}
-
-if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/badge-magic-android" -o "$TRAVIS_BRANCH" != "$DEPLOY_BRANCH" ]; then
-    echo "We decrypt key only for pushes to the master branch and not PRs. So, skip."
-    exit 0
-fi
```

Comment:
You've just given everyone in the world the ability to sign any app they want with the badge-magic signing key. Congrats

There's a reason why these checks were in place

Reply:
lol, should have removed only the deploy branch condition. Updated

Reply:
lol, should have changed only the deploy branch condition. Updated